src/test/java/com/acrolinx/client/sdk/integration/CheckTest.java

```diff
@@ -553,8 +558,8 @@ public void testCheckWithDocumentMetaData()
         }
     }
 
-    @Test(expected = AcrolinxException.class)
-    public void testCheckWithDocumentMetaDataAsList() throws AcrolinxException
+    @Test // (expected = AcrolinxException.class)
```

Comment:
TODO FIXME (there are a couple of out-commented `(expected = SomeException.class)` here and in other tests